src/_lib/sass/src/_popovers.scss

```diff
@@ -2,6 +2,7 @@
 // Popovers
 // --------------------------------------------------
 
+$popover-border-color: get-color('gray', 5x-light);
```

Comment:
Should we not just update the `$popover-border-color` in variables.scss?

Reply:
Yep.